### ISSUE TITLE
Can't load adapters from other config in console

### ISSUE THREAD

**Issue by Bibi0803** [frankframework/frankframework#5448]
**❗ please do not add sensitive information in issues, you can provide extra information via email using issue number as reference ❗**

**Describe the issue**
There are two adapters defined in one config as below
![image](https://github.com/ibissource/iaf/assets/60199670/7f76429f-68f9-4a80-b3b2-21e05aa732e1)
![image](https://github.com/ibissource/iaf/assets/60199670/77bd5e86-6a3d-4518-a81c-d95ba6a40629)

and there are other configs trying to load those two adapters by giving values in deploymentSpecifics.properties such as:
![image](https://github.com/ibissource/iaf/assets/60199670/8357b825-90c5-417d-a83a-63a76dd68d90)

in framework version 7.7.5, this works fine, and in console you will see those two adapters are loaded correctly with the defined name under each config, when upgrading to 7.8.1 version, they are not loaded in console Adapter Status any more, but you could still see the code with correct adapter name in Show Configurations -> Loaded Configuration, larva tests in those configs which are directly calling those two adapters work fine, but the pipeline name in Ladybug is "Pipeline ${FT_Rebranding_AdapterAndListener.name}" instead of actual name such as "ALFA_Rebranding". But the larva tests for other adapters calling those 2 adapters with IbisLocalSender failed with SenderException: could not find JavaListener ...
For example:
FT_Rebranding_Adapter.active=true
FT_Rebranding_AdapterAndListener.name=ENUT_Rebranding
FT_Rebranding_Receiver.name=ENUT_Rebranding_Receiver

![image](https://github.com/ibissource/iaf/assets/60199670/cecf4b25-8976-4180-a77d-03ce9de7769e)

Failed with "SenderException: could not find JavaListener [ENUT_Rebranding]"

**Reporter**
Bibi

**Environment**
```
FF! 7.8.1: XXX---
Running on LPCENL009517959 using Apache Tomcat/9.0.74
Java Version: OpenJDK Runtime Environment (1.8.0_292-b10)
Heap size: 302M, total JVM memory: 652M
Free disk space: 84GB, total disk space: 236GB
Up since: 2023-09-21 14:48:13 (1d) 
```

**Additional Environment**
 - Browser: Firefox

**Comment by nielsm5** [comment]
I noticed you're using XML entity resolvers, could you try using the 'framework' includes, as specified here:
https://github.com/ibissource/iaf/blob/master/test/src/main/configurations/HTTP/Configuration.xml

**Comment by Bibi0803** [comment]
'framework' includes is not available in 7.8.1 version

**Comment by nielsm5** [comment]
I cannot reproduce this, has the order been set correctly (first the parent, then implementing configurations)?
This is a very creative way of loading configurations, and seems to work in the 7.9 and 8.0.
Both with XML includes as well as the `<Include ref="config-file.xml"/>` element.

**Comment by Matthbo** [comment]
Looks like this issue was caused by #6020 and should be fixed when 8.0 comes out
